site/en/docs/webstore/cws-dashboard-listing/index.md

```diff
@@ -91,9 +91,8 @@ To explain how your extension works in more detail, you can provide a direct lin
 
 ### Providing a support URL {: #support-url }
 
-The CWS provides a built in user support system under your items' "Support tab", but you can use your own support page by including a link in the **Support URL**. Note that first you must activate "Enable User Feedback" in the Account settings page. 
-
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/ArMfJztL1OlP6UektUwb.png", alt="Enable User Feedback Support tab", width="800", height="185" %}
+The CWS provides a [built in user support][support-tab] experience under your items' **Support tab**, but you can use
```

Comment:
Ah, perhaps this is legacy but I don't think we ever say "CWS" officially. It should be "Chrome Web Store".